contents/docs/llm-analytics/installation/anthropic.mdx

```diff
@@ -12,6 +44,12 @@ npm install @anthropic-ai/sdk
 
 </MultiLanguage>
 
+<LLMsSDKsCallout />
+
+</Step>
+
+<Step title="Initialize PostHog and the Anthropic wrapper" badge="required">
+
 In the spot where you initialize the Anthropic SDK, import PostHog and our Anthropic wrapper, initialize PostHog with your project API key and host (from [your project settings](https://us.posthog.com/settings/project)), and pass it to our Anthropic wrapper.
```

Comment:
```suggestion
In the spot where you initialize the Anthropic SDK, import PostHog and our Anthropic wrapper, initialize PostHog with your project API key and host from [your project settings](https://us.posthog.com/settings/project), and pass it to our Anthropic wrapper.
```

Ditto to other lines similar to this. No need to do `()`, it's not an aside it's very relevant